src/set_theory/game/nim.lean

```diff
@@ -140,7 +140,7 @@ begin
   introI hG,
   rw [impartial.equiv_iff_sum_first_loses, ←impartial.no_good_left_moves_iff_first_loses],
   intro i,
-  equiv_rw left_moves_add G (nim (grundy_value G)) at i,
+  equiv_rw (@to_left_moves_add G (nim (grundy_value G))).symm at i,
```

Comment:
Can you make more arguments explicit and avoid the need for so many new `@`?

Reply:
Making both arguments explicit of course avoids the `@`, but conversely restores a bunch of underscores that had been removed elsewhere. Not sure if that trade is worth it.